### PR TITLE
Support dapr multirun from CLI.

### PR DESCRIPTION
**Pull request by artursouza**
Signed-off-by: Artur Souza <asouza.pro@gmail.com>

It works but I still get an error if I try to update CLI. There is a difference in version being used by runtime and CLI for k8s.io/client-go. We might need a new RC for CLI.

```
$ go get -u github.com/dapr/cli@v1.10.0-rc.3
go: downloading golang.org/x/net v0.6.0
github.com/dapr/cli imports
        github.com/dapr/cli/cmd imports
        github.com/dapr/cli/pkg/kubernetes imports
        k8s.io/client-go/plugin/pkg/client/auth/openstack: cannot find module providing package k8s.io/client-go/plugin/pkg/client/auth/openstack
```

### CODE REVIEW

**Comment by artursouza** [comment]
I verified it worked on standalone mode.

File: go.mod [hunk 1]
```diff
@@ -1,169 +1,152 @@
...
+	github.com/dapr/dapr v1.10.0-rc.5
 	github.com/gorilla/mux v1.8.0
-	github.com/gorilla/websocket v1.4.2
-	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
+	github.com/gorilla/websocket v1.5.0
+	github.com/stretchr/testify v1.8.1
+	k8s.io/api v0.26.1
```
Comment (mukundansundar) [inline go.mod:11]: Add a replace to use 0.25.2

**Review by mukundansundar — COMMENTED** [review]
@artursouza related to this https://github.com/dapr/cli/issues/1114

File: go.mod [hunk 2]
```diff
@@ -1,169 +1,152 @@
...
+	k8s.io/apimachinery v0.26.1
+	k8s.io/client-go v0.26.1
```
Comment (mukundansundar) [inline go.mod:13]: Add a replace to use 0.25.2

Comment (artursouza) [reply go.mod:13]: I cannot because `go mod tidy` reverts it back to the latest version.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**